source/common/runtime/BUILD

```diff
@@ -24,6 +24,7 @@ envoy_cc_library(
         "//source/common/common:hash_lib",
         "//source/common/singleton:const_singleton",
         "//source/common/singleton:threadsafe_singleton",
+        "//source/common/quic/platform:flags_impl_lib",
```

Comment:
fun fact, we allow compiling QUIC out and one of the CI bots (compile time options) will likely fail if this is direct included.

you'll probably need to wrap this with

] + envoy_select_enable_http3([
your library
]

and your code with
#ifdef ENVOY_ENABLE_QUIC
#endif
You can test this locally with building and testing with --//bazel:http3=False

Reply:
done